.formatter.exs

```diff
@@ -2,7 +2,6 @@
 [
   inputs:
     ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
-    |> Enum.flat_map(&Path.wildcard(&1, match_dot: true))
```

Comment:
What this does is expand the `["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]` patterns, so we can specify files to be formatted one by one, and reject those under `lib/proto` and `test/generated`.